commands/dns_cli.py

```diff
@@ -98,6 +98,9 @@ def create_cert(client, domain_client, domain, base_len):
         if hz["Name"] == domain_to_create:
             domain_id = hz["Id"]
             break
+    if not domain_id:
+        click.secho(f"Unable to find Domain ID for {domain_to_create} in the hosted zones")
```

Comment:
Do we want styling here? Maybe a red colour... If we do not want colour then we should be using `click.echo()` rather than `click.secho()`.

```suggestion
        click.secho(f"Unable to find Domain ID for {domain_to_create} in the hosted zones", fg="red")
```

Reply:
It is now.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/120/commits/2d0bcd9cceb3ed36678e238e1eef1c1a1bbcedbe.